### ISSUE TITLE
Fail to get ip and crash

### ISSUE THREAD

**Issue by daledude** [TrilliumIT/docker-arp-ipam#1]
Interesting driver!

I ran into an issue where it crashes. I have 8.2.250.40/24 assigned on a second interface. Below is the output of the app. Lines with "Is this available" are what I added to see if the ip's checked were available which many of them were and some were already assigned elsewhere.

I have another arp-ipam running on a 10.0.48.0/22 network successfully. This is assigned to the primary interface. I'm using 2 swarm servers.
docker-arp-ipam --addr 10.0.48.4:8181 --exclude-first 30 --debug --name arp-ipam2

```
root@container1:~# docker-arp-ipam --addr 8.2.250.40:8181 --exclude-first 101 --debug --name arp-ipam3
time="2017-10-05T12:20:45-07:00" level=info msg=Starting Version=0.27
time="2017-10-05T12:20:45-07:00" level=debug msg=NewDriver
time="2017-10-05T12:20:45-07:00" level=debug msg="Starting driver"
time="2017-10-05T12:20:55-07:00" level=debug msg="RequestPool: &{arp-ipam-default 8.2.250.0/24  map[] false}"
time="2017-10-05T12:20:55-07:00" level=debug msg="RequestAddress: &{8.2.250.0/24 8.2.250.3 map[RequestAddressType:com.docker.network.gateway]}"
time="2017-10-05T12:20:55-07:00" level=debug msg="Specific Address Requested: 8.2.250.3"
time="2017-10-05T12:20:55-07:00" level=debug msg="Gateway requested, approving"
time="2017-10-05T12:20:55-07:00" level=debug msg="RequestAddress served" Address=8.2.250.3/24 Time="672.946µs"
time="2017-10-05T12:21:10-07:00" level=debug msg="RequestAddress: &{8.2.250.0/24  map[]}"
time="2017-10-05T12:21:10-07:00" level=debug msg="Random Address Requested in network 8.2.250.0/24"
time="2017-10-05T12:21:10-07:00" level=debug msg="Generating Random Address in network 8.2.250.0/24"
time="2017-10-05T12:21:10-07:00" level=debug msg="Generating Random Address in network 8.2.250.0/24"
time="2017-10-05T12:21:10-07:00" level=debug msg="Is this available" IP=8.2.250.206
time="2017-10-05T12:21:10-07:00" level=debug msg="Generating Random Address in network 8.2.250.0/24"
time="2017-10-05T12:21:10-07:00" level=debug msg="Is this available" IP=8.2.250.109
time="2017-10-05T12:21:10-07:00" level=debug msg="Generating Random Address in network 8.2.250.0/24"
time="2017-10-05T12:21:10-07:00" level=debug msg="Is this available" IP=8.2.250.107
time="2017-10-05T12:21:10-07:00" level=debug msg="Is this available" IP=8.2.250.236
time="2017-10-05T12:21:10-07:00" level=debug msg="Generating Random Address in network 8.2.250.0/24"
time="2017-10-05T12:21:10-07:00" level=debug msg="Generating Random Address in network 8.2.250.0/24"
time="2017-10-05T12:21:10-07:00" level=debug msg="Is this available" IP=8.2.250.193
time="2017-10-05T12:21:10-07:00" level=debug msg="Generating Random Address in network 8.2.250.0/24"
time="2017-10-05T12:21:10-07:00" level=debug msg="Is this available" IP=8.2.250.22
time="2017-10-05T12:21:10-07:00" level=debug msg="Is this available" IP=8.2.250.125
time="2017-10-05T12:21:10-07:00" level=debug msg="Is this available" IP=8.2.250.167
time="2017-10-05T12:21:13-07:00" level=debug msg="Generating Random Address in network 8.2.250.0/24"
time="2017-10-05T12:21:13-07:00" level=debug msg="Generating Random Address in network 8.2.250.0/24"
time="2017-10-05T12:21:13-07:00" level=debug msg="Is this available" IP=8.2.250.12
time="2017-10-05T12:21:13-07:00" level=debug msg="Is this available" IP=8.2.250.228
time="2017-10-05T12:21:13-07:00" level=debug msg="Generating Random Address in network 8.2.250.0/24"
time="2017-10-05T12:21:13-07:00" level=debug msg="Is this available" IP=8.2.250.213
time="2017-10-05T12:21:13-07:00" level=debug msg="Is this available" IP=8.2.250.113
time="2017-10-05T12:21:16-07:00" level=debug msg="Generating Random Address in network 8.2.250.0/24"
time="2017-10-05T12:21:16-07:00" level=debug msg="Generating Random Address in network 8.2.250.0/24"
time="2017-10-05T12:21:16-07:00" level=debug msg="Generating Random Address in network 8.2.250.0/24"
time="2017-10-05T12:21:16-07:00" level=debug msg="Is this available" IP=8.2.250.3
time="2017-10-05T12:21:16-07:00" level=debug msg="Is this available" IP=8.2.250.77
time="2017-10-05T12:21:16-07:00" level=debug msg="Is this available" IP=8.2.250.51
time="2017-10-05T12:21:16-07:00" level=debug msg="Is this available" IP=8.2.250.15
time="2017-10-05T12:21:16-07:00" level=debug msg="Is this available" IP=8.2.250.146
time="2017-10-05T12:21:16-07:00" level=debug msg="Is this available" IP=8.2.250.144
time="2017-10-05T12:21:16-07:00" level=debug msg="Is this available" IP=8.2.250.53
time="2017-10-05T12:21:16-07:00" level=debug msg="Is this available" IP=8.2.250.191
panic: runtime error: invalid memory address or nil pointer dereference
[signal 0xb code=0x1 addr=0x10 pc=0x479283]

goroutine 9 [running]:
panic(0x880ce0, 0xc82000e110)
        /usr/lib/go-1.6/src/runtime/panic.go:481 +0x3e6
github.com/TrilliumIT/docker-arp-ipam/driver.(*neighSubscription).probeAndWait(0xc8200a5240, 0xc8200a9470, 0x1dcd65000, 0x0, 0x0, 0x0)
        /root/go/src/github.com/TrilliumIT/docker-arp-ipam/driver/address.go:125 +0x8e3
github.com/TrilliumIT/docker-arp-ipam/driver.getNewRandomUnusedAddr(0xc82001b8f0, 0x1dcd65000, 0xc8200a5240, 0x65, 0x0, 0xc820089500, 0x0, 0x0)
        /root/go/src/github.com/TrilliumIT/docker-arp-ipam/driver/suggest.go:191 +0x83b
github.com/TrilliumIT/docker-arp-ipam/driver.(*Driver).getRandomUnusedAddr(0xc8200a07c0, 0xc82001b8f0, 0x1dcd65000, 0x1, 0x0, 0x0)
        /root/go/src/github.com/TrilliumIT/docker-arp-ipam/driver/suggest.go:156 +0xfc
github.com/TrilliumIT/docker-arp-ipam/driver.(*Driver).requestAddress(0xc8200a07c0, 0xc82001b770, 0x0, 0x0, 0x0)
        /root/go/src/github.com/TrilliumIT/docker-arp-ipam/driver/ipam.go:221 +0xc75
github.com/TrilliumIT/docker-arp-ipam/driver.(*Driver).RequestAddress.func1(0xc8200a07c0, 0xc82001b770, 0xc82014c4e0, 0xc82014c540)
        /root/go/src/github.com/TrilliumIT/docker-arp-ipam/driver/ipam.go:157 +0x2f
created by github.com/TrilliumIT/docker-arp-ipam/driver.(*Driver).RequestAddress
        /root/go/src/github.com/TrilliumIT/docker-arp-ipam/driver/ipam.go:160 +0x10c
```

**Comment by clinta** [comment]
This should be fixed in 495fb4ae213df15a6f9f27a8aee2f2bc7bd9c5a0.

**Comment by daledude** [comment]
Thank you!
